### PR TITLE
Add GF(GoFrame) framework

### PR DESCRIPTION
**Pull request by johng-cn**
This PR adds another go framework `GF(GoFrame)` to this amazing project.

### CODE REVIEW

File: FRAMEWORKS.yml [hunk 1]
```diff
@@ -191,6 +191,10 @@ go:
     website: godoc.org/github.com/kataras/muxie
     version: "1.0"
     language: "1.11"
+  gf:
+    website: gfer.me
+    version: "1.1.994"
```
Comment (waghanza) [inline FRAMEWORKS.yml]: could you use the format : X.Y -> it will be more readable for our audience :+1:

**Comment by waghanza** [comment]
@johng-cn Could you use **go modules** ?

File: FRAMEWORKS.yml (hunk 1 above)
Comment (johng-cn) [reply FRAMEWORKS.yml]: ok, done

**Comment by johng-cn** [comment]
@waghanza yes, it used go modules, you can see it in `/go/gf/go.mod`

**Comment by waghanza** [comment]
@johng-cn should not `go.mod` have 2 requires ? as their is 2 imports in `main.go`

**Comment by johng-cn** [comment]
@waghanza As `GF` for this test codes, only one `require` in `go.mod` is neccessary, although there're two `import`s in `main.go`, but they're in the same package. You can see this's similar of `beego` framework in `/go/beego/go.mod` and `/go/beego/main.go`.

**Comment by waghanza** [comment]
@johng-cn I've update results https://github.com/the-benchmarker/web-frameworks#full-table :tada: 

You might be disapointed :stuck_out_tongue: results are not (yet) **production-ready**

**Comment by johng-cn** [comment]
@waghanza Thanks , I've seen the results. That's great, it means `GF` do need some performance improvement. Comparison makes progress. 😄

**Comment by johng-cn** [comment]
@waghanza Not yet, lots of work to do. I'll remember to tell you if such's done.

**Pull request merged**